doc/whats_new/_contributors.rst

```diff
@@ -176,4 +176,6 @@
 
 .. _Nicolas Hug: https://github.com/NicolasHug
 
-.. _Guillaume Lemaitre: https://github.com/glemaitre
+.. _Ashutosh Hathidara: https://github.com/ashutosh1919
```

Comment:
this is nowadays reserved for core devs whose names are entered frequently into the changelog. Usually we just use `:user:`. Please revert this.

Reply:
Done